### ISSUE TITLE
Target $I$ for the Gated-LSTM baseline

### ISSUE THREAD

**Issue by lionminhu** [facebookresearch/House3D#63]
Hello, I have been trying to implement the Gated-LSTM baseline according to the descriptions in the RoomNav paper ([link](https://arxiv.org/abs/1801.02209)), but I am confused with the some of the notations.

From Section 5.2.2 (for Gated-LSTM):
![image](https://user-images.githubusercontent.com/17691594/72311612-19f96800-363a-11ea-8bea-75ea52597f5d.png)

The symbol `$I$` is used to denote the target instruction. This same symbol was used in Section 5.1.2 (for Gated-CNN) to denote the target instruction *prior to* conversion to an embedding vector `$y$`:
![image](https://user-images.githubusercontent.com/17691594/72311693-5f1d9a00-363a-11ea-82b8-792eabfd9280.png)

So if we were to follow the same notation as in Section 5.1.2, we would have to feed the one-hot vector `$I$` (and not the embedding vector) as input to the gated fusion module and the LSTM for the Gated-LSTM.

However, the Figure 2 seems to imply that it is in fact the embedding vector that is fed into the gated fusion and the LSTM:
![image](https://user-images.githubusercontent.com/17691594/72311742-8b391b00-363a-11ea-8650-663b2fa599f0.png)

Could you clarify whether the target `$I$` for Gated-LSTM baseline is meant to be the one-hot vector or the embedding vector?

Thank you in advance.

**Comment by GoingMyWay** [comment]
@lionminhu Hi, where did you get the SUNCG data to install this library?

**Comment by lionminhu** [comment]
Previously (as in https://github.com/minosworld/minos/issues/94), we were able to request the authors for the SUNCG dataset.
Due to legal issues ([citation](https://futurism.com/tech-suing-facebook-princeton-data)), however, the download links and the forms have all been removed (https://github.com/facebookresearch/House3D/commit/c72733109165308b4fc716c0ed05805a38f0ca81).
I do not think the dataset is going to be available again in a while, if at all.

Relevant: https://github.com/facebookresearch/House3D/issues/57, https://github.com/minosworld/minos/issues/159

**Comment by GoingMyWay** [comment]
> Previously (as in [minosworld/minos#94](https://github.com/minosworld/minos/issues/94)), we were able to request the authors for the SUNCG dataset.
> Due to legal issues ([citation](https://futurism.com/tech-suing-facebook-princeton-data)), however, the download links and the forms have all been removed ([c727331](https://github.com/facebookresearch/House3D/commit/c72733109165308b4fc716c0ed05805a38f0ca81)).
> I do not think the dataset is going to be available again in a while, if at all.
> 
> Relevant: #57, [minosworld/minos#159](https://github.com/minosworld/minos/issues/159)

Yes, can you share the link for downloading the data or do you know how to use other datasets to fix this issue?

**Comment by lionminhu** [comment]
Unfortunately, I am not aware of any download links that work anymore.
My knowledge is possibly outdated, but if house dataset/environment is what you need, you may also want to consider [GibsonEnv](https://github.com/StanfordVL/GibsonEnv) with Matterport3D dataset.
